core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -945,13 +1093,37 @@ void core_mmu_get_user_va_range(vaddr_t *base, size_t *size)
 		*size = BIT64(L1_XLAT_ADDRESS_SHIFT);
 }
 
+static uint64_t *core_mmu_get_user_mapping_entry(struct mmu_partition *prtn,
+						 unsigned int base_idx)
+{
+#if (CORE_MMU_BASE_TABLE_LEVEL == 0)
+	uint8_t idx = 0;
+	uint64_t *tbl = NULL;
+#endif
+
+	assert(user_va_idx != -1);
+
+#if (CORE_MMU_BASE_TABLE_LEVEL == 0)
+	idx = prtn->user_l1_table_idx[base_idx][get_core_pos()];
+	tbl = prtn->xlat_tables[idx];
+
+	return &tbl[user_va_idx];
+#else
+	return &prtn->base_tables[base_idx][get_core_pos()][user_va_idx];
+#endif
+}
+
 bool core_mmu_user_mapping_is_active(void)
 {
-	bool ret;
+	bool ret = false;
 	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
+	uint64_t *entry = NULL;
 
 	assert(user_va_idx != -1);
```

Comment:
Can remove this test since needed only from `core_mmu_get_user_mapping_entry()` and already asserted there.